rt/transports/http/src/main/java/org/apache/cxf/transport/http/Headers.java

```diff
@@ -588,9 +589,10 @@ public static String toHttpDate(Date date) {
     public static String toHttpLanguage(Locale locale) {
         StringBuilder sb = new StringBuilder();
         sb.append(locale.getLanguage());
-        if (locale.getCountry() != null) {
+        String country = locale.getCountry();
+        if (!StringUtils.isEmpty(country)) {
```

Comment:
what about `Locale.toString().replace('_','-');`?

Reply:
Yes!  That is much cleaner.  Thanks!